model.py

```diff
@@ -9497,21 +9540,56 @@ def for_externalintegration(cls, key, externalintegration):
         return cls.for_library_and_externalintegration(
             _db, key, None, externalintegration
         )
-    
+
+    @classmethod
+    def _cache_insert(cls, setting, cache):
+        """Cache a ConfigurationSetting for later retrieval, probably by a
+        different database session.
+        """
+        library_id = setting.library_id
+        external_integration_id = setting.external_integration_id
+        key = setting.key
+        cache_key = (library_id, external_integration_id, key)
+        cache[cache_key] = setting
```

Comment:
Do you need to override _cache_insert here? Won't HasFullTableCache._cache_insert work since you've defined _cache_key?